lib/hydra/works/models/concerns/generic_file/characterization.rb

```diff
@@ -0,0 +1,116 @@
+module Hydra::Works::GenericFile
+  module Characterization
+    extend ActiveSupport::Concern
+
+    autoload :FitsDatastream, 'hydra/works/models/generic_file/characterization/fits_datastream.rb'
```

Comment:
If you extend ActiveSupport::Autoload, then you don't need the second arguments on any of these calls.